src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -279,6 +284,37 @@ public function addClassResource(\ReflectionClass $class)
     }
 
     /**
+     * Checks whether the requested class, interface or trait exists and registers the result for resource tracking.
+     *
+     * @param string $class
+     *
+     * @return bool
+     *
+     * @final
+     */
+    public function getClassExists($class)
```

Comment:
I'm not sure this is the best name for this method. What about `registerClass()` or `trackClass()` or something like that.